meteor/server/api/showStyles.ts

```diff
@@ -210,5 +223,8 @@ class ServerShowStylesAPI extends MethodContextAPI implements NewShowStylesAPI {
 	async reorderAllShowStyleVariants(showStyleBaseId: ShowStyleBaseId, orderedVariants: ShowStyleVariant[]) {
 		return reorderAllShowStyleVariants(this, showStyleBaseId, orderedVariants)
 	}
+	async renameShowStyleVariant(showStyleBaseId: ShowStyleBaseId, renamedShowStyleVariant: ShowStyleVariant) {
```

Comment:
This doesn't need to take a full `ShowStyleVariant`. It would be better to either have the variant id and name as separate parameters, or make it be a `renamedShowStyleVariant: Pick<ShowStyleVariant, '_id' | 'name'>`

Reply:
Good point! It has been changed!

Reply:
Just be aware that you don't end up with functions like this
```
function doSomething(id: string, name: string, configuration: SomeConfigurationType) {...}

function startDoingSomething(variant: ShowStyleVariant) {
  doSomething(variant.id, variant.name, variant.configuration)
}
```
where it could just be
```
function doSomething(showStyleVariant: ShowStyleVariant) {...}

function startDoingSomething(variant: ShowStyleVariant) {
  doSomething(variant)
}
```